lite/Dockerfile

```diff
@@ -56,6 +56,25 @@ RUN git clone --branch 2.0.2-2 --depth 1 "$GIT_SERVER/nanox-secure-sdk.git" "$NA
 ENV NANOSP_SDK=/opt/nanosplus-secure-sdk
 RUN git clone --branch 1.0.4 --depth 1 "$GIT_SERVER/nanosplus-secure-sdk.git" "$NANOSP_SDK"
 
+# Unified SDK
+ENV LEDGER_SECURE_SDK=/opt/ledger-secure-sdk
+RUN git clone "$GIT_SERVER/ledger-secure-sdk.git" "$LEDGER_SECURE_SDK"
```

Comment:
`API_LEVEL_0_SDK` ? `API_LEVEL_NONE_SDK` ?
Defining also a worktree for this one would ease the building scripts with almost 0 overhead

Reply:
I'm not sure what would be the use case for this one? But yeah, there won't be any overhead so I will add it!

Reply:
Eventually I'm not sure, as this is not tracked by tags, but only `master` branch this will lead to difference between each build, and as I don't find much usage for this, I think this is bringing more danger than good.